tests/mongo-tools/export_import_test.go

```diff
@@ -0,0 +1,119 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package mongotools
+
+import (
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestExportImport(t *testing.T) {
+	t.Parallel()
+
+	containerSourceRoot := "/dumps/mongodb-sample-databases/"
+	containerTestsRoot := "/dumps/tests"
+	localTestsRoot := filepath.Join("..", "..", "dumps", "tests")
```

Comment:
Thanks to the usage of the testname in the database name it's easy to store both mongorestore and mongodump tests inside the same directory

Reply:
It seems it's not, will fix that. But all other things can be reviewed